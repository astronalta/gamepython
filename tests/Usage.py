import os

os.system(os.path.join("..", "bin", "jogo") + " --help")
os.system(os.path.join("..", "bin", "jogo") + " -h")
