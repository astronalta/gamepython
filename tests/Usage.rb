system("../bin/apollo --help")
system("../bin/apollo -h")
