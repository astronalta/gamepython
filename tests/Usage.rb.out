Usage: ../bin/apollo [OPTIONS] FILE...

   -a, --assembly       Skip link stage and output assembly.
   -d, --dump FILE      Dump the AST to FILE.
   -h, --help           Print this help message.
   -l, --library FILE   Generate a shared library and write it to FILE.
   -o, --output FILE    Write compiler output to FILE.
   -p, --path DIR       Add the directory DIR to the source search path.

Usage: ../bin/apollo [OPTIONS] FILE...

   -a, --assembly       Skip link stage and output assembly.
   -d, --dump FILE      Dump the AST to FILE.
   -h, --help           Print this help message.
   -l, --library FILE   Generate a shared library and write it to FILE.
   -o, --output FILE    Write compiler output to FILE.
   -p, --path DIR       Add the directory DIR to the source search path.

